docker-compose.override.yaml

```diff
@@ -11,11 +11,19 @@ services:
         scripts/wait-for/wait-for ffc-demo-claim-message-queue:8161 --
         npm run migrate &&
         npm run start:watch"
+    ports:
+      - "3003:3003"
     depends_on:
       - ffc-demo-claim-message-queue
     networks:
       - ffc-demo-claim
 
+  ffc-demo-claim-postgres:
+    networks:
+      - ffc-demo-claim
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+

```

Comment:
We reversed similar changes to volumes in the user service because there is a benefit in creating volumes for tests - it allows data to be inspected after a failed test run to help work out what caused the failure. Since the test script now cleans up volumes before each run (and after successful runs) there is no longer the problem of volumes conflicting between test runs.

Reply:
I've put the volume back in